docfx/docfx.json

```diff
@@ -13,7 +13,8 @@
       "disableGitFeatures": false,
       "disableDefaultFilter": false,
       "noRestore": false,
-      "namespaceLayout": "flattened"
+      "namespaceLayout": "flattened",
+      "EnumSortOrder": "declaringOrder"
```

Comment:
This new setting replaces the `ManagedReference.extension.js` extension we have for sorting enumerators